pkg/generators/openapi.go

```diff
@@ -353,10 +353,49 @@ func (g openAPITypeWriter) generateCall(t *types.Type) error {
 	return g.Error()
 }
 
+func (g openAPITypeWriter) generateValueValidations(vs *ValueValidations) {
+
+	if vs == nil {
+		return
+	}
+	if vs.Minimum != nil {
+		g.Do("Minimum: util.PointerToFloat64($.Minimum$),\n", vs)
```

Comment:
`util` here may not be added correctly to imports. May have to support it to the generator args with `vs` and invoke the `|raw` filter on it which ensures to qualify the package correctly, and add it to imports.